src/idiff/idiff.cpp

```diff
@@ -418,7 +419,7 @@ main(int argc, char* argv[])
     }
 
     imagecache->invalidate_all(true);
-    ImageCache::destroy(imagecache);
+    imagecache.reset();
```

Comment:
I'm guessing the call to reset is to force the destruction to happen before shutdown as opposed to implicitly on end of scope? Maybe leave a comment about this, otherwise it looks like a line that could be removed.

Reply:
In fact - since this is for the _shared_ ImageCache object, the real "last" owner of the image cache is the global variable, so if I am thinking about this correctly, this line doesn't really accomplish much?

Reply:
yes, and will do